ui/imports/shared/views/AmountToReceive.qml

```diff
@@ -0,0 +1,75 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+ColumnLayout {
+    id: root
+
+    property var store
+    property var selectedAsset
+    property bool isLoading: false
+    property string amountToReceive
+    property bool isBridgeTx: false
+    property bool cryptoFiatFlipped: false
+
+    QtObject {
+        id: d
+        readonly property string zeroString: LocaleUtils.numberToLocaleString(0, 2)
+        readonly property string fiatValue: {
+            if(!!root.selectedAsset && !!amountToReceive) {
+                let cryptoValue = root.store.getFiatValue(amountToReceive, root.selectedAsset.symbol, root.store.currentCurrency)
+                return parseFloat(cryptoValue) === 0 ? LocaleUtils.numberToLocaleString(parseFloat(cryptoValue), 2) : LocaleUtils.numberToLocaleString(parseFloat(cryptoValue))
+            }
+            return zeroString
+        }
+    }
+
+    StatusBaseText {
+        Layout.alignment: Qt.AlignRight | Qt.AlignTop
+        text: root.isBridgeTx ? qsTr("Amount Bridged") : qsTr("Recipient will get")
+        font.pixelSize: 13
+        lineHeight: 18
+        lineHeightMode: Text.FixedHeight
+        color: Theme.palette.directColor1
+    }
+    RowLayout {
+        Layout.alignment: Qt.AlignRight
+        Layout.preferredHeight: 42
+        StatusBaseText {
+            Layout.alignment: Qt.AlignVCenter
+            id: amountToReceiveText
+            text: isLoading ? "..." : cryptoFiatFlipped ? d.fiatValue: amountToReceive
+            font.pixelSize: Utils.getFontSizeBasedOnLetters(text)
```

Comment:
If the purpose is to keep the text in a certain width, [`fontSizeMode`](https://doc.qt.io/qt-5/qml-qtquick-text.html#fontSizeMode-prop) with value `Text.HorizontalFit` looks like a simpler solution. What do you think?

Reply:
Hey so I am trying to acommodate the new design mentioned here. https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=9770%3A130516&t=64lZkdafLUz7UdQU-4

The reasoning for this function is to ensure the letters in the TextEdit resize as per design.
I could use the FontMetrics of the sizeModel in this componet but I need it to be have exactly the same as the TextEdit and hence the common function between the too.

I will rename it, good idea :) @stefandunca

Reply:
> Hey so I am trying to acommodate the new design mentioned here.

Thanks, @Khushboo-dev-cpp for clarifications, I missed this detail.